lib/trema.rb

```diff
@@ -0,0 +1,4 @@
+require 'trema/controller'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.